README.md

```diff
@@ -8,3 +8,17 @@ It covers:
   -   Add files to be tracked by Git `git add 'my-file'`.
   -   Stage changes to be commited `git add 'my-file'`.
   -   Commit changes with appropriate commit message `git commit -m 'commit message'`.
+  - Finding out information regarding what files are modified and those that are in the staging area using `git status`.
+  - Printing out all the commits that are completed using `git log`.
+  - Pushing all code from local repository to the remote repository on GitHub using `git push -u origin master`
+  -  Using `git pull origin master` to pull the latest changes from the remote repository to local
+  - Cloning existing repositories to your PC using `git clone [repository url]`.
+  
+
+
+# _A brief introduction to Git_
```

Comment:
This section is great. This should be our starter section, give this a slightly higher level heading and give it a bit of more content, brief and straight to the point

Reply:
Moving the introduction paragraph to the top of the readme